main.go

```diff
@@ -182,18 +181,11 @@ func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 	name := mux.Vars(r)["name"]
 	plan := r.FormValue("plan")
 
-	helmTargetArtifactTypeValue := "Helm"
-	yamlsTargetArtifactTypeValue := "Yamls"
-	knativeTargetArtifactTypeValue := "Knative"
-
 	re := regexp.MustCompile(`artifacttype:\\s.*`)
 	artifactTypematch := re.FindString(plan)
-	artifactType := yamlsTargetArtifactTypeValue
-	if strings.HasSuffix(artifactTypematch, knativeTargetArtifactTypeValue) {
-		artifactType = knativeTargetArtifactTypeValue
-	} else if strings.HasSuffix(artifactTypematch, helmTargetArtifactTypeValue) {
-		artifactType = helmTargetArtifactTypeValue
-	}
+	match := []rune(artifactTypematch)
+	artifactType := string(match[16:len(match)])
```

Comment:
Instead of hardcoding 16, can we ectract out the non spaced string from suffiix? By say using a regex? What do you think?

Reply:
Hi, 16 is the length of the former part of the string, it has nothing to do with the suffix. So if you want me to remove all possible whitespace, I can try to make the change. Or maybe you prefer to use regex groups to avoid such hardcoding like 16?

Reply:
Yes, Using regex groups would be a easier approach.